composer.json

```diff
@@ -13,12 +13,13 @@
         }
     ],
     "require": {
-        "php": ">=5.3.9",
+        "php": ">=5.5.0",
```

Comment:
But php 5.5 is almost dead and won't receive any updates really soon. "Forcing" an update to at ;east 5.6 (the ones stuck at < 5.6 can still use previous versions, that's all) is IMO the best factor to motivate upgrades...

Otherwise, let'sjust support 5.2 as there are still 10% usage :trollface: